### DISCUSSION TITLE
Add previous GSoC application proposals in our documentation of Mentored Projects

### DISCUSSION THREAD

**Discussion by MridulS** [networkx/networkx#5111] [General]
We have participated in GSoC three times and it would be great to collect more resources for future participants in our [documentation for mentored projects](https://networkx.org/documentation/latest/developer/projects.html#mentored-projects).

So just wanted to ping previous participants and see if they are interested in "open sourcing" the proposal for future participants in the program.

- GSoC 2011 @bjedwards (https://www.cs.unm.edu/~bedwards/gsoc1.html ?)
- GSoC 2015 @MridulS  (That's me!, and yes I am happy to add the proposal to the documentation)
- GSoC 2015 @OrkoHunter 
- GSoC 2021 @z3y50n 
- GSoC 2021 @harshal-dupare 
- GSoC 2021 @mjschwenne 
- GSoC 2021 @vdshk 

It would be a good reference for future GSoC/Outreachy applicants to look at these proposals to get an idea about expectations about writing a proposal for these mentored projects with NetworkX.

We can also add a quick paragraph about the work done/any other links to external blog posts et al.

PS: Let me know if I missed anyone or if there were other such contributors to NetworkX (outside of GSoC too).

**Comment by z3y50n** [comment]
Sure I would be happy to share my [proposal](https://docs.google.com/document/d/1efZ5bIbJc0bmWjucwKxTz6U_e1xc6N2H/edit?usp=sharing&ouid=106697604333970942550&rtpof=true&sd=true).

**Comment by OrkoHunter** [comment]
Awesome Mridul. Here's mine https://github.com/saketkc/fos-proposals/blob/master/GSoC-2015/Accepted/HimanshuMishra-PSF-Implementing-Add-On-System-For-NetworkX/GSoC-2015-Application-Himanshu-Mishra-Add-on-System-for-NetworkX.md

**Comment by mjschwenne** [comment]
I'm happy to share.

Here is a link to my [GSoC proposal](https://drive.google.com/file/d/1XGrjupLYWioz-Nf8Vp63AeuBVApdkwSa/view).
